src/components/LeftContainer.js

```diff
@@ -12,41 +12,55 @@ import 'codemirror/theme/dracula.css';
 import 'codemirror/mode/clike/clike';
```

Comment:
@Vipu97  Please update your code from the main branch before moving to the next comments